crates/ibc/src/applications/transfer/amount.rs

```diff
@@ -9,7 +9,7 @@ use crate::bigint::U256;
 #[derive(
```

Comment:
Please feel free to add a changelog entry. See here for details on how to do so -> https://github.com/cosmos/ibc-rs/blob/main/CONTRIBUTING.md#changelog.